src/scripts/main.js

```diff
@@ -354,7 +354,49 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.map((person) => {
+  // creating rows for every person
 
-// write your code here
+  const createRows = document.createElement('tr');
+
+  // creating cells
+
+  const cellName = document.createElement('td');
+
+  cellName.textContent = person.name;
+  createRows.insertAdjacentElement('beforeend', cellName);
+
+  const cellGender = document.createElement('td');
+
+  if (person.sex === 'f') {
+    cellGender.textContent = 'Female';
+  } else {
+    cellGender.textContent = 'Male';
+  }
+
+  createRows.insertAdjacentElement('beforeend', cellGender);
+
+  const cellBorn = document.createElement('td');
+
+  cellBorn.textContent = person.born;
+  createRows.insertAdjacentElement('beforeend', cellBorn);
+
```

Comment:
Try to use innerHTML for simplify your logic https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML
```suggestion
  const createRows = document.createElement('tr');

  createRows.innerHTML = `
    <td>.....</td>
    ...........
  `

```